### PR TITLE
Add more control over settings

### PR DESCRIPTION
**Pull request by ZwerOxotnik**
There are no errors in logs.

### CODE REVIEW

File: game_settings.lua [hunk 1]
```diff
@@ -1,4 +1,8 @@
 return {
     game_mode = 2,
-    last_winner_name = "[[WINNER_NAME]]"
+    last_winner_name = "[[WINNER_NAME]]",
+    mode_name = "Medium", -- Doesn't affect gameplay
+    age_score_factor = 0.8, -- Other values before this commit: 5.0, 0.25
+    research_evo_score_factor = 65, -- Other values before this commit: 100, 65
+    tech_price_multiplier = 0.35 -- Other values before this commit: 0.2, 0.75 (affects game.difficulty_settings.technology_price_multiplier during init)
```
Comment (blubFisch) [inline game_settings.lua:7]: But we don't want to tweak this for every round, it will be confusing for players... I think they should become stable

Comment (ZwerOxotnik) [reply game_settings.lua:7]: It's easier maintain, adapt, test in such way. The point is easier customization. It's fine to try other values etc., especially with mods.

Comment (ZwerOxotnik) [reply game_settings.lua:7]: Also, players not really understand this in the first place.

**Comment by blubFisch** [comment]
Sorry, I'd rather have the existing system of pre-set mode settings, to be more predictable for players. Let's talk about changes before you make a PR next time

**Pull request closed without merging**